core/testdata/TestJSONSerialisation_VersionedSignedProposal.json#01.golden

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
If we add this, do we still need to keep the golden blinded equivalent? Also, probably it would be nice to include `"blinded": false` in the non-`#01` version.

Reply:
hmm well there's no value so it'll be false by default 

> keep the golden blinded equivalent

you mean the golden files for VersionedSignedBlindedProposal?